Resources/views/Gallery/index.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-{% sonata_template_box 'This is the gallery index template. Feel free to override it.' %}
+{% sonata_template_box 'This is the gallery index template. Feel free to override it.' 'SonataMediaBundle'%}
```

Comment:
Well on https://github.com/sonata-project/SonataMediaBundle/blob/3.x/Resources/views/Block/block_gallery.html.twig#L19 it is already used also with Twig 2.0 the following appears in the debug toolbar:
<img width="1071" alt="screen shot 2017-05-14 at 8 38 10 pm" src="https://cloud.githubusercontent.com/assets/720487/26036389/67a17098-38e5-11e7-8d5d-06634b7935ce.png">